.github/labeler.yml

```
@@ -49,6 +49,11 @@
   - "templates/setup.sh.in"
   - "views/**/*"
 
+"topic: magnets":
+  - "compact/fields/**/*"
+  - "compact/fields.xml"
+  - "compact/solenoid.xml"
```

This may be fragile since far backward and far forward might change their xml interfaces, but should we add this?
```suggestion
  - "compact/solenoid.xml"
  - "compact/far_backward/magnets.xml"
  - "compact/far_forward/magnets.xml"
```

---

We do